src/ducks/apps/components/Sections/components/DropdownFilter.jsx

```diff
@@ -2,12 +2,13 @@ import React, { Component } from 'react'
 
 import SelectBox from 'cozy-ui/transpiled/react/SelectBox'
 import Icon from 'cozy-ui/transpiled/react/Icon'
+import Bottom from 'cozy-ui/transpiled/react/Icons/Bottom'
 import PropTypes from 'prop-types'
 
 const SmallArrow = () => (
   <Icon
     className="sto-sections-dropdown-icon"
-    icon="bottom"
+    icon={Bottom}
```

Comment:
this change is not tested, can we try adding one test? 🙏

Reply:
I'm not sure why we want to test a graphic item 🤔 

Or maybe what you want is a snapshot test?